packages/nx/src/utils/params.ts

```diff
@@ -922,8 +922,10 @@ async function promptForValues(
     .prompt(getPromptsForSchema(opts, schema, projectsConfigurations))
     .then((values) => ({ ...opts, ...values }))
     .catch((e) => {
-      console.error(e);
-      process.exit(0);
+      console.error(
+        `Could not properly parse prompts. Error:\n${e.message || e}`
+      );
+      process.exit(1);
```

Comment:
We don't know for certain that it would be a parsing error. It could be that, or it could also be an `enquirer` specific error. Let's keep the previous, more generic error logging, and only change the exit code.
```suggestion
      console.error(e);
      process.exit(1);
```